src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_TupleLiteralExpression.vb

```diff
@@ -0,0 +1,101 @@
+                If smallestCtor Is Nothing Then
+                    Return New BoundBadExpression(
+                                    syntax,
+                                    LookupResultKind.Empty,
+                                    ImmutableArray(Of Symbol).Empty,
+                                    StaticCast(Of BoundNode).From(rewrittenArguments),
+                                    type,
+                                    hasErrors:=True)
+                End If
+
+                Dim smallestConstructor As MethodSymbol = smallestCtor.AsMember(smallestType)
+                Dim currentCreation As BoundObjectCreationExpression = New BoundObjectCreationExpression(syntax, smallestConstructor, smallestCtorArguments, initializerOpt:=Nothing, type:=smallestType)
+
+                If underlyingTupleTypeChain.Count > 0 Then
+                    Dim tuple8Type As NamedTypeSymbol = underlyingTupleTypeChain.Peek()
+                    Dim tuple8Ctor As MethodSymbol = DirectCast(TupleTypeSymbol.GetWellKnownMemberInType(tuple8Type.OriginalDefinition,
+                                                                                            TupleTypeSymbol.GetTupleCtor(TupleTypeSymbol.RestPosition),
+                                                                                            _diagnostics,
+                                                                                            syntax), MethodSymbol)
+                    If tuple8Ctor Is Nothing Then
+                        Return New BoundBadExpression(
+                                    syntax,
+                                    LookupResultKind.Empty,
+                                    ImmutableArray(Of Symbol).Empty,
+                                    StaticCast(Of BoundNode).From(rewrittenArguments),
+                                    type,
+                                    hasErrors:=True)
+                    End If
+
+                    ' make successively larger creation expressions containing the previous one
+                    Do
+                        Dim ctorArguments As ImmutableArray(Of BoundExpression) = ImmutableArray.Create(rewrittenArguments,
+                                                                                              (underlyingTupleTypeChain.Count - 1) * (TupleTypeSymbol.RestPosition - 1),
+                                                                                              TupleTypeSymbol.RestPosition - 1).Add(currentCreation)
+
+                        Dim constructor As MethodSymbol = tuple8Ctor.AsMember(underlyingTupleTypeChain.Pop())
+                        currentCreation = New BoundObjectCreationExpression(syntax, constructor, ctorArguments, initializerOpt:=Nothing, type:=constructor.ContainingType)
+
+                    Loop While (underlyingTupleTypeChain.Count > 0)
+
+                End If
+
+                'currentCreation = currentCreation.Update(
```

Comment:
This should be just removed. I missed that

---
In reply to: [71414998](https://github.com/dotnet/roslyn/pull/12596#discussion_r71414998) [](ancestors = 71414998)